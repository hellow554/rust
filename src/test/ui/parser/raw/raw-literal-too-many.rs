fn main() {
    let foo = r##"bar"####; //~ERROR too many `#` when terminating raw string
}
