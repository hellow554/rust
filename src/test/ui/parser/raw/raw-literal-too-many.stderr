error: too many `#` when terminating raw string
  --> $DIR/raw-literal-too-many.rs:2:15
   |
LL |     let foo = r##"bar"####;
   |               ^--^^^^^----
   |                |      |
   |                |      ...but is closed with 4.
   |                this raw string has 2 leading `#`...
   = help: remove the unneeded `#`

error: aborting due to previous error

